sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
+        if "vti" in str(device):
             status += "Device: %s " % device.split("STRING: ")[1].rstrip()
             status += "%s\n" % device_status[index].split("INTEGER: ")[1]
 
-    if "down" in status:
+    if "down" in status or "2" in status:
```

Comment:
I think we should only check for integer "2" since that is what is reliably returned by SNMP. A comment would probably be good explaining that "2" means "down".